usermods/Internal_Temperature_v2/readme.md

```diff
@@ -0,0 +1,11 @@
+# MCU Temp Usermod
+This usermod adds the temperature readout to the Info tab and also publishes that over the topic `mcutemp` topic.
+
+
+A shown temp of 53,33°C might indicate that the internal temp is not supported.
+
+ESP8266 does not have a internal temp sensor
+
+ESP32S2 seems to crash on reading the sensor -> disabled
+
+Add a build flag `-D USERMOD_INTERNAL_TEMPERATURE` to your platformio environment.
```

Comment:
There should be some autor information here, plus some info about who is maintaining the usermod.

Please discuss with @lost-hope on what would be appropriate.

Reply:
@softhack007 perhaps you have a template or an example of how it should look like?

@lost-hope any thoughts about this?

Reply:
If in doubt put both authors in. Possibly with email or Github nickname & profile link.

Reply:
Done